homeassistant/components/barry/config_flow.py

```diff
@@ -0,0 +1,90 @@
+"""Adds config flow for Barry integration."""
+# pylint: disable=attribute-defined-outside-init
+from pybarry import Barry, InvalidToken
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_ACCESS_TOKEN
+
+from .const import DOMAIN  # pylint:disable=unused-import
```

Comment:
No need to disable this pylint check

```suggestion
from .const import DOMAIN
```